cfme/fixtures/pytest_selenium.py

```diff
@@ -1043,7 +1043,14 @@ def force_navigate(page_name, _tries=0, *args, **kwargs):
 
     def _login_func():
         if not current_user:  # default to admin user
-            login.login_admin()
+            if(store.current_appliance.mode == "ssui"):
+                print "-----------"
+                print store.current_appliance.mode
+                login.ssui_login()
+            else:
+                print "-----------"
+                print store.current_appliance.mode
+                login.login_admin()
```

Comment:
I see the linter complaining about the print statements, it is much mure practical to use logger in these cases.